pulsar/internal/pulsartracing/readme.md

```diff
@@ -0,0 +1,40 @@
+### Usage
```

Comment:
@maraiskruger1980 thanks for your contribution! Want users to know this great feature and know how to use it? 

If so, could you please add docs to [Pulsar - Go client](https://pulsar.apache.org/docs/en/next/client-libraries-go/)?
(How about adding a section "Monitoring" and add the docs under that section? Explanations of "What is the feature" & "How to use the feature" are needed. You can take [How to trace Pulsar messages with OpenTracing and Jaeger](https://streamnative.io/en/blog/tech/2020-06-11-opentracing-instrumentation-for-pulsar/) and [OpenTracing Pulsar Client](https://hub.streamnative.io/monitoring/opentracing-pulsar-client/0.1.0) as references. 

Then I would like to help review. Feel free to ping me if needed, thanks.